### PR TITLE
login-screen optimizations

### PR DESCRIPTION
**Pull request by lduer**
contribution to #483 

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [x] My code follows the code style
- [ ] All files have a license header 
- [ ] All methods have a doc header with type declarations
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes

### CODE REVIEW

**Comment by kevinpapst** [comment]
Thanks! 👍 

That worked for sure in the past, but I might have changed the theme-bundle and forgot to adapt it in Kimai.

File: templates/bundles/FOSUserBundle/Security/login.html.twig [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {% extends '@AdminLTE/FOSUserBundle/Security/login.html.twig' %}
 
-{% block logo_login %}<b>Kimai</b><br>TimeTracking{% endblock %}
+{% block logo_login %}{% include 'partials/logo_login.html.twig' %}{% endblock %}
```
Comment (kevinpapst) [inline templates/bundles/FOSUserBundle/Security/login.html.twig]: @lduer Is there a reason for the difference between `logo_login.html.twig` and `login_logo.html.twig`?
Was that by intention and a file is missing or is it a bug? See comment: https://github.com/tobybatch/kimai2/issues/5#issuecomment-456747207

Comment (kevinpapst) [reply templates/bundles/FOSUserBundle/Security/login.html.twig]: Clarified, see #520

Comment (lduer) [reply templates/bundles/FOSUserBundle/Security/login.html.twig]: Thanks for fixing :) obviously a typo

**Pull request merged**